sql/004_search.sql

```diff
@@ -218,8 +218,8 @@ prop_path := replace(att_parts.dotpath, 'properties.', '');
 op := CASE _op
     WHEN 'eq' THEN '='
     WHEN 'ge' THEN '>='
-    WHEN 'gt' THEN '>'
-    WHEN 'le' THEN '<='
+    WHEN 'gte' THEN '>'
```

Comment:
@bitner  this will break the pgstac backend of fast-api, no? Because of [these operators](https://github.com/stac-utils/stac-fastapi/blob/master/stac_fastapi/pgstac/stac_fastapi/pgstac/types/search.py#L25-L27)?

Reply:
yes. the backend is pegged to 0.2.4, so will need to update stac-fastapi pgstac with this and change the pin to 0.2.5

Reply:
hi @bitner and @lossyrob: I ended up on this PR as I was encountering the issue - thanks for fixing it! I think the greater than's got mixed up. Should be the following to match the change happening with the less than's:
```sql
WHEN 'gt' THEN '>'
WHEN 'gte' THEN '>='
```
Should I open a PR to fix this?

Reply:
Yes! Great catch...a PR would be appreciated, thank you!

Reply:
@lossyrob, will you make me a collaborator so I can open the PR? Fixed the issue but don't have access. I imagine I may contribute in the future anyhow

Reply:
Ah, the best way to make a PR would be to fork the project into your GitHub account, clone that repo (or add your fork as a remote, e.g. `git remote add mine git@github.com:/zstatmanweil/pgstac`, push the branch to your fork (`git push mine my-fix-branch`), and then make the PR from their to the main stac-utils/pgstac GitHub repository. Would that work for you? You can make PRs against this project in that way, and also any other open source project even if your not a collaborator.

Reply:
Got it - thanks for the explanation. PR opened!